Resources/Maps/_CD/Shuttles/syndicate-diplomat.yml

```diff
@@ -0,0 +1,5518 @@
+meta:
+  format: 7
+  category: Grid
+<<<<<<< HEAD
```

Comment:
To add onto Aquif's mention, your map file itself is riddled with the same markers too.